environments/production/app-processes.yml

```diff
@@ -93,7 +93,7 @@ celery_processes:
       num_workers: 2
     malt_generation_queue:
       pooling: gevent
-      concurrency: 40
+      concurrency: 20
```

Comment:
7 machines * 40 * 2 = 560 tasks running at a time.
7 machines * 20 * 2 = 280 tasks running at a time. That's still probably too much and maybe more than the total tasks? I might suggest
```
      concurrency: 3
      num_workers: 1  # this is the default so you can also just exclude the property
```
for a total concurrency of 21.

Although the problem may have to do more with the individual queries than how many are running at the same time, so if that's the case then even this lower concurrency won't help and the fix has to be made in the code.

Reply:
Cal pointed out to me that I'm wrong and that this is running on one single machine not 7. So the numbers are just 80 => 40. Thanks Cal